packages/ui-progress/src/ProgressBar/__examples__/ProgressBar.examples.tsx

```diff
@@ -31,28 +32,27 @@ export default {
     valueNow: [0, 40, 80, 100],
     renderValue: [
       null,
-      // @ts-expect-error ts-migrate(7031) FIXME: Binding element 'valueNow' implicitly has an 'any'... Remove this comment to see the full error message
-      // eslint-disable-next-line react/display-name
-      ({ valueNow, valueMax }) => (
-        <span>{Math.round((valueNow / valueMax) * 100)}</span>
-      )
+      ({
+        valueNow,
+        valueMax
+      }: {
+        valueNow: ProgressBarProps['valueNow']
+        valueMax: ProgressBarProps['valueMax']
+      }) => <span>{Math.round((valueNow! / valueMax!) * 100)}</span>
```

Comment:
You can use the `Values` type here so that you don't need the `!`-s

Reply:
Darn, missed this. The Values idea was an afterthought, thanks for catching it